modules/javafx.controls/src/test/java/test/com/sun/javafx/scene/control/infrastructure/ControlSkinFactory.java

```diff
@@ -282,7 +281,7 @@ public static void attemptGC(WeakReference<?> weakRef) {
      * @return the list of the data converted to one-dimensional arrays
      */
     public static List<Object[]> asArrays(List<?> data) {
-        List<Object[]> result =  (List) data.stream()
+        List<Object[]> result =  data.stream()
                 .map(d -> new Object[] {d, })
                 .collect(toList());
         return result;
```

Comment:
According to the javadoc on this method, it's a hack for old Junit versions. Maybe it isn't needed anymore even. Leave as is for this PR anyway.

Reply:
Sorry, that's not clear, remove the cast or revert this?  Whatever that comment says, I seriously doubt this cast would have any effect on how old JUnit's would behave unless someone is mocking things they shouldn't be (ie. someone mocked the `data` parameter).

Reply:
Sorry, remove the cast. The javadoc says
```
     * Nasty hack to keep JUnit pre-4.12 happy.
     * Before 4.12, Parameterized can only handle
     * a collection of arrays.
```

I think that the whole method could be removed, but it might not be worth dealing with this, especially not now.